tutorials/pages/create-nft-with-iota-sdk.md

```diff
@@ -0,0 +1,229 @@
+
+```javascript
+async function uploadByPath(filePath) {
+  try {
+    console.log('\n');
+    console.log(consoleColor, `Start local IPFS node for upload:`);
+
+    // Set up local IPFS node for upload
+    let node;
+    if (!node) {
+      node = await IPFS.create({
+        repo: `ipfs_node`,
+      });
+    }
+
+    // Read file from path
+    const file = fs.readFileSync(filePath);
+
+    // Upload file to IPFS
+    const fileAdded = await node.add(file);
+    const contentIdentifier = fileAdded.path;
+
+    console.log('\n');
+    console.log(
+      consoleColor,
+      `Your file was uploaded to IPFS with the following Content Identifier (CID):`,
+    );
+    console.log(contentIdentifier, '\n');
+
+    console.log(consoleColor, `Check your file on IPFS:`);
+    console.log(`https://ipfs.io/ipfs/${contentIdentifier}`, '\n');
+
+    return contentIdentifier;
+  } catch (error) {
+    console.error('IPFS upload error', error);
+  }
+}
+```
+
+#### 3. Prepare the NFT's metadata
+
+This part calls the `uploadByPath()` function described above and prepares the metadata of your NFT. Make sure to give your NFT a nice name here.
+
+```javascript
+async function run() {
+    try {
+        const filePath = "nft-image.jpg";
+        const ipfsCid = await uploadByPath(filePath);
+
+        // Define NFT metadata
+        const metadataObject = {
+            standard: "IRC27",
+            type: "image/jpeg",
+            version: "v1.0",
+            name: "<Enter_your_desired_name_here>",
+            uri: `https://ipfs.io/ipfs/${ipfsCid}`
+        };
+
+        const metadataBytes = utf8ToHex(JSON.stringify(metadataObject));
+```
+
+#### 4. Import Stronghold account and synchronize
+
+```javascript
+if (!process.env.STRONGHOLD_PASSWORD) {
+  throw new Error('.env STRONGHOLD_PASSWORD is undefined, see .env.example');
+}
+
+const wallet = new Wallet({
+  storagePath: process.env.WALLET_DB_PATH,
+});
+
+const account = await wallet.getAccount('Alice');
+
+// We send from the first address in the account.
+const senderAddress = (await account.addresses())[0].address;
+
+// We need to unlock stronghold.
+await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
+```
+
+#### 5. Mint NFT
+
+We've to prepare our params for NFT minting, which are defined in MintNftParmas type.
+- `address`: (optional) Bech32 encoded address to which the Nft will be minted. Default will use the first address of the account.
+- `sender`: Bech32 encoded sender address
+- `metadata`: Hex encoded bytes containing metadata as per IRC27 from [TIP-27](https://wiki.iota.org/shimmer/tips/tips/TIP-0027/#nft-schema) as done in previous step.
+- `tag`: Hex encoded bytes with some custom tags that can help you categorise.
+- `issuer`: Bech32 encoded issuer address
+- `immutableMetadata`: Similar to metadata as hex encoded bytes however this one can not be updated later once minted. Can contain core properties that is supposed to be immutable.
```

Comment:
```suggestion
You should prepare the parameters for minting the NFT and define them in a `MintNftParms` type.
- `address`: (optional) Bech32 encoded address to which the Nft will be minted. The default will use the first address of the account.
- `sender`: Bech32 encoded sender address
- `metadata`: Hex encoded bytes containing metadata as per IRC27 from [TIP-27](https://wiki.iota.org/shimmer/tips/tips/TIP-0027/#nft-schema) as done in the previous step.
- `tag`: Hex encoded bytes with custom tags that can help you categorize.
- `issuer`: Bech32 encoded issuer address
- `immutableMetadata`: Similar to metadata as hex encoded bytes. However, this one can not be updated later once minted. It can contain core properties that are immutable.
```